src/dcap_provider.cpp

```diff
@@ -896,6 +901,17 @@ static void pck_cert_url(
     url << API_VERSION_07_2021;
 }
 
+static stringstream build_cache_url(const std::string& qe_id, const std::string& cpu_svn, const std::string& pce_svn, const std::string& pce_id) {

```

Comment:
Thanks for extracting this, it will make easier to make the cache keys consistent